microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -28,19 +28,29 @@ In Windows 11, Progressive Web Apps (PWAs) can define widgets, update them, and
 
 Widgets are experimentally supported starting with Microsoft Edge 108.
 
-To enable widget support in Microsoft Edge for your website:
+#### Enable local development of widgets
 
-1. Enroll your site in the [Web App Widgets](https://developer.microsoft.com/microsoft-edge/origin-trials/web-app-widgets/registration/) origin trial. To learn more about origin trials, see [how to enroll your site in an origin trial](origin-trials.md#enroll-your-site-in-an-origin-trial).
+To enable local development of widgets:
+
+1. Install [WinAppSDK 1.2](/windows/apps/windows-app-sdk/older-downloads#windows-app-sdk-12).
 
 1. In Microsoft Edge, go to `edge://flags`.
 
-1. In the **Search flags** text field, type **Web App widgets**.
+1. In the **Search flags** text field, type **Web App widgets**, and in the **Web App widgets** dropdown menu, select **Enabled**.
```

Comment:
```suggestion
1. In the **Search flags** text box, type **Web App widgets**, and then in the **Web App widgets** dropdown menu, select **Enabled**.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/text-box -- allows "text box", not "field".